### ISSUE TITLE
Recreate Special:AllPages and other special pages?

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython.github.io#85]
The old MediaWiki page had http://biopython.org/wiki/Special:AllPages in the sidebar which was useful when browsing the content, and for ensuring search engines found everything. For now we can browse the source at https://github.com/biopython/biopython.github.io/tree/master/wiki

This can probably be done quite easily with the same liquid templating as used on the recreated wiki tag/category pages via https://github.com/biopython/biopython.github.io/blob/master/_layouts/tagpage.html

See also https://www.mediawiki.org/wiki/Special:SpecialPages - there may be others worth resurrecting.

Note we'd want to white-list ``wiki/Special:AllPages`` again in ``robots.txt``

**Comment by MarkusPiotrowski** [comment]
This touches another problem: There is a number of sites that are 'unreachable', i.e. they are not linked from within `biopython.org`. Here is a list I made approx. one week ago:

|Files not being linked from within `biopython.org`|
|-------------------------------------------------------------|
|64-bit_Windows_Biopython                                 |
|Ace_contig_class.md                                            |
|Category_People.md                                            |
|Degenerated_Codons.md|
|File_talk_Contig_class.png.md                             |
|Gene_Ontology.md                                               |
|GitMigration.md|
|MotifDev.md|
|PDBParser.md|
|PopGen_dev_Statistics.md|
|ProtParam.md|
|Struct.md|
|Subversion_migration.md|
|User_Alexl.md|
|User_Blivens.md|
|User_Davidw.md|
|User_Edward.Tao.md|
|User_Gotgenes.md|
|User_Jblucks.md|
|User_MarcSaric.md|
|User_Orbeckst.md|
|User_ShawnDouglas.md|

Some of these are (outdated?) drafts (e.g. `PDBParser`), some may be linked as 'cookbook' entries (e.g. `Degenerated_Codons`) others as usual 'wiki' entries (e.g. `Gene_Ontology`). We have already discusse the `User_XXX` pages here #61

**Comment by peterjc** [comment]
Yes, this is what the related special page ``wiki/Special:LonelyPages`` was for.

All these 'hidden' pages should be reviewed (separate issue), e.g. I've removed ``File_talk_Contig_class.png.md`` with https://github.com/biopython/biopython.github.io/commit/ca5845c3ad1bf330546eca3a735556ed273f4eaf (I deliberately didn't include the talk pages for normal wiki and user pages) and in https://github.com/biopython/biopython.github.io/commit/6a729829c317ae8f225c396781cd28133321fc2b you added ``Degenerated_Codons.md`` to the cookbook category.
